public/views/bruno.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<h1>Bruno pernadas</h1>
+<body>
+    <p>Bruno Jorge de Oliveira Pernadas (born 9 December 1982 / Lisbon) is a Portuguese Musician, Composer, Arranger and Producer. He has a degree in Music by the Superior School of Music of Lisbon (ESML) in Jazz department.</p>
+</body>
+<link rel="stylesheet" href="app.js">
+<a href="work.html"></a>
```

Comment:
Here you don't need to link your app.js file, plus the a tag has no text inside and it's outside of the <body> tag meaning that it will never show up on your HTML